### ISSUE TITLE
Simplisafe Outage?

### ISSUE THREAD

**Issue by virtualjad** [nzapponi/homebridge-simplisafe3#43]
Anyone having trouble getting to simplisafe.com? Web and app are both suddenly down for me...and hoping it's a complete coincidence.

**Comment by murphmr** [comment]
Yes also having an issue - seems to be related to my internet provider though - if I try from cellular it works fine but just not over my internet provider. Who do you have? AT&T?

**Comment by virtualjad** [comment]
Same here... connected to ISP (Verizon FiOS) I'm getting 403 Forbidden. Works from cell (AT&T). I am able to get to https://webapp.simplisafe.com/ but can't log in.

Needless to say this makes controlling via Homebridge / Homekit impossible. I wish there was a way to access the devices locally.

**Comment by nikonratm** [comment]
Doubt you guys are having the same issue.
403 is an authentication issue, I've gotten it in the past with the plugin and seemed like my IP was [temporarily] getting blocked, I stopped the plugin / Homebridge and left it for a few hours and once I verified I could login again via website / app tried again.

See: https://github.com/nzapponi/homebridge-simplisafe3/issues/12
https://github.com/nzapponi/homebridge-simplisafe3/issues/31

Not being able to reach the website or app sounds more like ISP, etc. Again, if you cant login to website / app, its not a plugin issue.

**Comment by nzapponi** [comment]
Same issue here... I can’t access SimpliSafe either!

**Comment by virtualjad** [comment]
I guess I was flagged and was being blocked for some reason (likely the API storm that was coming from HB and HomeKit). I was eventually able to renew my external IP and got through. 

I’m curious what their threshold is. Might open a ticket with SS to log it.

**Comment by nzapponi** [comment]
Strange we all got it at the same time...

**Comment by nikonratm** [comment]
Still curious if was all the same thing, @nzapponi and @virtualjad you guys were testing out the smart lock update, maybe all that that got you guys temporarily banned (hence 403) but OP couldn’t even reach the website...

**Comment by kaicherry** [comment]
I am gonna "co-sign" this; I have access to SimpliSafe over cell but not from my network. Looks like a block to me at the ip level.

**Comment by murphmr** [comment]
I can get to the site again. Stopped homebridge for a few hours. 
Pretty sure it was tied to my IP address, I didn’t get the 403 when I went through a proxy.

**Comment by nikonratm** [comment]
Oh I forgot to mention, as was in the other issues linked to above, if this keeps happening consider dropping your sensor refresh timer to 30s, default is 15. Eg config:
`"sensorRefresh": 30`
I had to do that ages ago cause I would get blocked once or twice a day, since I set it to 30 never happens.

I wonder if SS were/are experimenting with more aggressive rate-limiting. hope not, it’s bad enough to flake out on HomeKit support but this would be really uncool...
